### PR TITLE
Various CSS + Link Fixes

### PR DESCRIPTION
**Pull request by pieromqwerty**
Fixes to the blog, including:
- Used a different arrow for the "Read More" and "Back to Blog" links for greater continuity across devices.
- Slightly brightened links for better legibility (especially on dark mode)
- Removed dead links from the footer
- Updated links in the footer to better match links across the rest of the site

### CODE REVIEW

**Comment by pieromqwerty** [comment]
Oh yeah, also removed a google analytics script which was hanging around in one of the ejs files for some reason. Is 100% gone now.

**Review by sergeichestakov — COMMENTED** [review]
sweet! can you post screenshots of before & after in both light and dark mode pls?

**Comment by pieromqwerty** [comment]
Lemme get those rq

**Comment by pieromqwerty** [comment]
Arrow Before:
image
image

Arrow After:
image
image

These screenshots also show the links

File: static/style.css [hunk 1]
```diff
@@ -4,7 +4,8 @@
   --text-color-diff: black;
   --bg-color: white;
     
-  --primary-color: #0070E0;
+  /*--primary-color: #0070E0;*/
```
Comment (sergeichestakov) [inline static/style.css:7]: think you can just remove, no need to comment out since it's saved in git

File: views/footer.ejs [hunk 2]
```diff
@@ -45,65 +44,56 @@
...
             <a href="https://replit.com/languages/apl">APL</a>
             <a href="https://replit.com/languages/lua">Lua</a>
-            <a href="https://replit.com/languages/python">Python 2.7</a>
             <a href="https://replit.com/languages/ruby">Ruby</a>
             <a href="https://replit.com/languages/roy">Roy</a>
-            <a href="https://replit.com/languages/python3">Python</a>
-            <a href="https://replit.com/languages/nodejs">Nodejs</a>
-            <a href="https://replit.com/languages/enzyme">Enzyme</a>
-            <a href="https://replit.com/languages/go">Go</a>
+            <a href="https://replit.com/languages/python">Python</a>
```
Comment (sergeichestakov) [inline views/footer.ejs:61]: FYI this link 404s - this should be python3
```suggestion
            <a href="https://replit.com/languages/python3">Python</a>
```

File: views/post-page.ejs [hunk 3]
```diff
@@ -41,25 +41,9 @@
...
-                    }),
-                    (i[r].l = 1 * new Date());
-                (a = s.createElement(o)), (m = s.getElementsByTagName(o)[0]);
-                a.async = 1;
-                a.src = g;
-                m.parentNode.insertBefore(a, m);
-            })(window, document, "script", "https://www.google-analytics.com/analytics.js", "ga");
-            ga("create", "UA-25629695-1", "auto");
-            ga("send", "pageview");
-        </script>
```
Comment (sergeichestakov) [inline views/post-page.ejs:62]: why are you removing Google Analytics?

**Review by sergeichestakov — COMMENTED** [review]
Awesome, thanks! Couple quick comments

File: views/post-page.ejs (hunk 3 above)
Comment (pieromqwerty) [reply views/post-page.ejs:62]: We meant to remove it long ago from the blog, most of the scripts were removed already. Patrick has more details as to why.

**Review by sergeichestakov — APPROVED** [review]
Thank you! 💅

**Pull request merged**